environmentsetup/initial-.com-environment-setup.md

```diff
@@ -0,0 +1,67 @@
+1.	Sign in using your work email and associated password.
+1.	Under the **settings** (looks like a gear) dropdown select **Security**. 
+1.	Select **GTM Sportswear – Development Group** in the drop down list on the left hand side of the page.
+1.	Select **Members** and verify that your name is included on the list of members.
+    
+### Clone the Project
+
+Now Open Visual studio 2017. In the top right hand corner of Visual Studio is a prompt to sign in as a user. 
+1.	Sign in using your work email and password.
+1.	If your account has been given proper permissions you should now be able to access GTM projects through the **Team Explorer** window. 
+1.	Select the **green connection icon** at the top of the **Team Explorer** window.
+1.	Click the **manage connections Icon** and select **Connect to a Project** in the dropdown.   
+1.	From the new view you can select the **gtm ->> gtm** project and clone it to a destination of your choice (the default location is fine).
+1.  Once the project has finished downloading, it will be automatically selected and appear in the **Solution Explorer**.
+1.  Double click on **GTMSportswear** in the **Solution Explorer**. This sets it as the default startup project.
+1.  Visual Studio may ask you if you want to update the Typescript version in your project to match the current version. Select **no**. We do not transpile typescript through Visual Studio, we instead use seperate build tools (setup below) through Window's Command Prompt.
+
+### Remove Gtm.Web from Visual Studios Build List
+
+Next you will want to remove **Gtm.Web** from the list of projects to build while debugging. This is because **Gtm.Web** has not been configured to ignore Typescript files while building in Visual Studio, resulting in build errors. 
+1.	In Visual Studio go to **Build ->>  Configuration Manager** (Build is a tab along the top of Visual Studio).
+1.	Deselect build for **Gtm.Web**.
+
+### Change Window's Setting
+
+By default, Windows may prevent Visual Studio’s emulator from displaying the website through localhost. This problem can be fixed by changing a setting through Window’s control panel.
+1.	Go to **Control Panel ->> Programs ->> Programs and Features ->> Turn Windows features on or off ->> Internet Information Services**.
+1.	Activate **Internet Information Services**.
+    
+### Install Web Dependancies
+
+The GTMSportswear project has several web dependancies that are not tracked through source control. This is because they can be easily installed in your local environment with npm. 
+1. Open Window's Command Prompt and change the working directory to **\<gtm_repository\>\GTMSportswear.com**. 
+1. Run **npm install**.
+1. If Grunt is not installed, run **npm install grunt --save-dev**.
+1. Install the Grunt command line interface globally with **npm install -g grunt-cli**.
+1. Install the Karma command line interface globally with **npm install -g karma-cli**.
+   
+### Run the Emulator

```

Comment:
I don't know the exact terminology, but I don't think "Emulator" is the word we want. It feels like I more often hear this as "Run in Debug Mode" or "Run on Localhost", something like that. Not a big deal of course!

Reply:
Changed